tests/test_firefox_urls.py

```diff
@@ -37,6 +37,11 @@ class TestFirefoxURLs(object):
         '/firefox/tiles/',
         '/firefox/unsupported-systems/',
         '/firefox/unsupported/EOL/',
+        # Legacy URLs
+        '/firefox/start/central.html',
+        '/firefox/en-US/firefox/whatsnew/',
+        '/firefox/panorama/',
+        'firefox/sync/firstrun.html',
     ]
 
     def test_urls_ok(self, mozwebqa):
```

Comment:
Should we also be checking that these urls don't involve more than one redirect? Could just `assert len(response.history) < 2`. I'm thinking it might catch issues with errant apache configs.